test/Tests/Strings/AutomatonTests.cs

```diff
@@ -2264,7 +2279,7 @@ public void DeterminizationPreservesProperties()
                 determinizableAutomaton,
                 x =>
                 {
-                    x.TryDeterminize(out var determinized);
+                    var determinized = x.TryDeterminize();
```

Comment:
You could make this code block significantly simpler now:  x => x.TryDeterminize()

Reply:
Indeed.